api/client/node/inspect.go

```diff
@@ -26,10 +25,13 @@ func newInspectCommand(dockerCli *client.DockerCli) *cobra.Command {
 	var opts inspectOptions
 
 	cmd := &cobra.Command{
-		Use:   "inspect [OPTIONS] self|NODE [NODE...]",
-		Short: "Display detailed information on one or more nodes",
-		Args:  cli.RequiresMinArgs(1),
+		Use:   "inspect [OPTIONS] [NODE...]",
+		Short: "Display detailed information on one or more nodes (default to current node)",
+		Args:  nil,
```

Comment:
I'm gonna be removing this shortly as it's covered in another PR by someone else. The only logic that will be in this one is the `-a` flag stuff.